plugins/modules/content_upload.py

```diff
@@ -17,11 +17,13 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from __future__ import absolute_import, division, print_function
+
 __metaclass__ = type
 
 
 DOCUMENTATION = '''
 ---
+Note that the OSTree content upload is not idempotent - running mutliple times will attempt to upload the content unit.
```

Comment:
this would need to go in the `notes` section (line 60-ish)

`DOCUMENTATION` is  YAML